listenbrainz/webserver/views/api.py

```diff
@@ -665,6 +665,64 @@ def get_service_details(user_name):
     return jsonify({'user_name': user_name, 'services': services})
 
 
+@api_bp.route("/tags/", methods=['GET', 'OPTIONS'])
+@crossdomain
+@ratelimit()
+def get_tags_dataset():
+    """ Get the recordings with the specified tag. """
+    tag = request.args.getlist("tag")
+    if tag is None:
+        raise APIBadRequest("tag param is missing")
+
+    condition = request.args.get("condition")
+
+    # if there is only one tag, then we can use any of the condition's query to retrieve data
+    if len(tag) == 1 and condition is None:
+        condition = "OR"
+
+    if condition is None:
+        raise APIBadRequest("multiple tags are specified but the condition param is missing")
+    condition = condition.upper()
+    if condition != "AND" and condition != "OR":
+        raise APIBadRequest("condition param should be either 'AND' or 'OR'")
+
+    try:
```

Comment:
I think for an API we should use percent values from 0 - 100 -- it little more intuitive for people, I think.

Reply:
done